docs/writing-tests.md

```diff
@@ -300,7 +314,13 @@ it('some test', async ({ browser }) => {
 Parameters:
 
 - state (required) `String` – state name; should be unique within one test
-- selector (required) `String|String[]` – DOM-node selector that you need to capture
+- selector (optional) `String|String[]` – DOM-node selector that you need to capture. If not specified or skipped, will be set to `body` and the following options will be automatically added to opts: 
+```{
+    allowViewportOverflow: true,
```

Comment:
Why it is `true` by default? To my mind it should be set the same as for `assertView` by user selector

Reply:
I don't see the answer. To my mind we don't need to specify it.